picard/ui/mainwindow.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why is this line removed ?

Reply:
@zas It is a BOM(Byte Order Mark) and BOMs are not preferred for UTF-8 files. So, @phw and @Freso  asked me to remove it.

Reply:
Ah, sorry. That was a misunderstanding derstanding. The BOM are just three bytes at the very beginning, before the `#`. They are not visible here, this makes the discussion a bit difficult. But it go added maybe by your editor automatically. It's maybe even still in the file, despite you removed the first line.

So `# -*- coding: utf-8 -*-` should be added again. And before saving check your text editor if there is some option regarding the BOM. What editor are you using?

And to not clutter this pull request with too many commits I would suggest when doing these changes you ammend the last commit with the BOM message. That way it stays only a single commit. You have to force push the to GitHub, but for a feature branch that's ok (but never do this on a project's master branch, it will cause trouble).

Reply:
Done.